client/src/app.html

```diff
@@ -4,6 +4,26 @@
 		<meta charset="utf-8" />
 		<meta name="description" content="" />
 		<link rel="icon" href="%svelte.assets%/favicon.png" />
+		<link rel="stylesheet" href="/smui.css" media="(prefers-color-scheme: light)" />
+		<link rel="stylesheet" href="/smui-dark.css" media="screen and (prefers-color-scheme: dark)" />
+		<!-- Material Icons -->
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/icon?family=Material+Icons"
+			crossorigin="anonymous"
```

Comment:
VScode was issuing warning shots because of it and the details it wrote made sense to me at that time, otherwise probably not really, but they can't hurt can it?

Reply:
Sounds fine.